sdk/storage/azfile/internal/shared/shared.go

```diff
@@ -62,6 +62,13 @@ const (
 	FileAttributesDirectory = "Directory"
 )
 
+const (
+	ServiceClient   = "service.Client"
```

Comment:
Can you please update to `azcore@v1.6.1` and then add the prefix `azfile/` to all of these constants?  At present, this will generate the wrong package name `service` in the telemetry string.

Reply:
Done